### ISSUE TITLE
safescale cluster expand command blocks after image selection by safescaled

### ISSUE THREAD

**Issue by wruffine-csgroup** [CS-SI/SafeScale#114]
## Version used
Safescaled version: 21.11.0, build 62f078347ba043e3c88f4ab2af7672a911b655ff (2021/12/23-16:02), with Tags: (release,)

## Bug description
When one uses the command `safescale cluster expand <my-cluster-name>`, safescaled doesn't return any log after the image choice, as if the daemon was stuck right after this step.
```
TRACE   [2022-01-04 12:58:54.054][trace][ 273651] >>> {task e70c946d-7fd9-4781-8d9f-43ed73e3726b/cluster/demo/expand} listeners.(*ClusterListener).Expand('demo') [.../go/src/github.com/CS-SI/SafeScale/lib/server/listeners/cluster.go:382] 
TRACE   [2022-01-04 12:58:54.208][trace][ 273651] >>>  operations.(*Cluster).GetFlavor() [.../go/src/github.com/CS-SI/SafeScale/lib/server/resources/operations/cluster.go:452] 
TRACE   [2022-01-04 12:58:54.208][trace][ 273651] <<<  operations.(*Cluster).GetFlavor() [.../go/src/github.com/CS-SI/SafeScale/lib/server/resources/operations/cluster.go:458] 
TRACE   [2022-01-04 12:58:54.208][trace][ 273651] >>> {task e70c946d-7fd9-4781-8d9f-43ed73e3726b/cluster/demo/expand} operations.(*Cluster).AddNodes(1) [.../go/src/github.com/CS-SI/SafeScale/lib/server/resources/operations/cluster.go:1102] 
TRACE   [2022-01-04 12:58:58.876][trace][ 273651]                                             os-image (OSIMAGE                                              ): WagnerFischerScore:   0 
INFO    [2022-01-04 12:58:58.877][info][ 273651] Selected image: 'os-image' (ID='xxxxxxxxxxxxxxxx')
```
No new node is created and the safescale command doesn't return.

**Comment by vplanche-cssi** [comment]
Fixed in commit 9ba8d37f0878d6b2f2a73f8b270df27ef98af9d1.

Will be included in next release v21.11.1 (but not yet planned, may take a couple of weeks).

You may need to compile your own binaries from the branch release/v21.11.

**Comment by oscarpicas** [comment]
In order to test the fix, you should build the branch 'develop' or wait for next release. Assuming you have git, and docker in your machine, the fastest way to build is:
 
```
git clone https://github.com/CS-SI/SafeScale -depth 1 -b develop
cd SafeScale
./create-local-docker.sh
```
 
After that, you should see a ./exported directory, in there you should found the safescale and safescaled binaries
